frontend/app/public/index.html

```diff
@@ -23,6 +23,9 @@
       content="width=device-width, initial-scale=1, shrink-to-fit=no"
     />
     <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.png" />
+    <link rel="preload" href="../src/assets/fonts/SourceSansPro/SourceSansPro-Regular.woff2" as="font" type="font/woff2" crossorigin>
+    <link rel="preload" href="../src/assets/fonts/SourceSansPro/SourceSansPro-SemiBold.woff2" as="font" type="font/woff2" crossorigin>
+    <link rel="preload" href="../src/assets/fonts/SourceSansPro/SourceSansPro-Bold.woff2" as="font" type="font/woff2" crossorigin>
```

Comment:
Hm, I'm not sure if these relative paths will work when the frontend is compiled (it'd be worth manually testing these changes with a built wheel file because don't think CI will catch the case where these links fail to preload). In case that they do, I think we'll still want to load these relative to `%PUBLIC_URL%` just because it feels a bit cleaner.

The paths relative to `%PUBLIC_URL%` will look something like `%PUBLIC_URL%/static/media/font.hash.woff2`. The hash is included by webpack and is annoying to have to worry about but necessary. We may want to think of some sort of way to detect if the hashes change and break this preloading, though, since it seems possible that we might update our fonts in the future and not realize the hashes in our `index.html` file no longer correspond to those in the compiled static assets.

Reply:
Appreciate the help - I've updated to paths relative to `%PUBLIC_URL%`  & added a FE test checking these hashes